libs/wingc/src/lsp/sync.rs

```diff
@@ -107,14 +105,9 @@ fn partial_compile(source_file: &str, text: &[u8], jsii_types: &mut TypeSystem)
 
 	let type_diag = type_check(&mut scope, &mut types, &Path::new(source_file), jsii_types);
 
-	// Analyze inflight captures
-	let mut capture_visitor = CaptureVisitor::new();
-	capture_visitor.visit_scope(&scope);
```

Comment:
I guess we're missing some diagnostics now here now? Are we? Maybe open an issue and run this by @MarkMcCulloh?

Reply:
I don't think there should be any missing diagnostics, or if they should they should appear in our snapshots.